src/components/Hotspot.tsx

```diff
@@ -11,20 +11,21 @@ const defaultContainerStyles = {
 };
 
 const defaultButtonsStyles = {
-  position: 'relative',
+  position: 'absolute',
```

Comment:
this would position the button according to the first position ancestor. which problem does this change fix ?

Reply:
mmmm....
i don't know what the design here is, but this sounds like a case where a tooltip should be used